From 51029ebc94e6cda68ad5c64b2d732b0c305ae201 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Thu, 6 Dec 2012 16:48:33 -0800
Subject: [PATCH 166/795] MR1: CDH-9220. Fix TestStreamingStatus to check log contains error

Reason: Test failure
Ref: CDH-9220
Author: Karthik Kambatla
(cherry picked from commit 42e0089c14b802bfe88c5ce956ae48e2bc9c3f3c)
(cherry picked from commit d31b9c7a9acd043d127a5954c5d8fd6972ac1bae)
---
 .../hadoop/streaming/TestStreamingStatus.java      |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java b/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java
index dc80c9c..6c0a416 100644
--- a/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java
+++ b/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java
@@ -307,7 +307,7 @@ void validateTaskStderr(StreamJob job, TaskType type)
 
     // trim() is called on expectedStderr here because the method
     // MapReduceTestUtil.readTaskLog() returns trimmed String.
-    assertTrue(log.equals(expectedStderr.trim()));
+    assertTrue(log.contains(expectedStderr.trim()));
   }
 
   // Validate if user counter is incremented properly
-- 
1.7.0.4

